ui/raidboss/data/03-hw/raid/a10s.ts

```diff
@@ -122,26 +134,29 @@ export default {
       netRegexCn: NetRegexes.ability({ source: '佣兵雷姆普里克斯', id: '1AB[89AB]' }),
       netRegexKo: NetRegexes.ability({ source: '용병 레임브릭스', id: '1AB[89AB]' }),
       preRun: (data, matches) => {
-        data.charges = data.charges || [];
-        data.charges.push({
+        const charges = data.charges ??= [];
+        const chargeMap: { [abilityId: string]: string } = {
           '1AB8': 'getIn',
           '1AB9': 'getOut',
           '1ABA': 'spread',
           '1ABB': 'stackMarker',
-        }[matches.id]);
+        };
+        charges.push(chargeMap[matches.id] ?? 'unknown');
```

Comment:
How does this get back to data.charges?

Reply:
My intention was that `charges === data.charges` here and so `charges.push` will change both of them.  Is that not true?

Reply:
Oh, I misread line 137. Understood.